spirv_msl.cpp

```diff
@@ -309,6 +325,11 @@ uint32_t CompilerMSL::add_interface_block(StorageClass storage)
 		break;
 	}
 
+	case StorageClassUniformConstant:
+	{
+		ib_var_ref = stage_uniform_var_name;
```

Comment:
OMG such a stupid mistake. I'm ashamed. :S I will add a break and hopefully never forget it again.